### PR TITLE
remove cloud event extensions duplicated data more effective

### PR DESCRIPTION
**Pull request by zcong1993**
Signed-off-by: zcong1993 <zhangcong1992@gmail.com>

# Description

Use `set` to remove cloud event extensions duplicated data more more effective.

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/runtime/runtime.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ import (
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
+	"github.com/dapr/kit/logger"
```
Comment (berndverst) [inline pkg/runtime/runtime.go:33]: this import has to be moved next to the other github.com/dapr imports or the linter will fail.

**Review by berndverst — CHANGES_REQUESTED** [review]

File: pkg/runtime/runtime.go (hunk 1 above)
Comment (zcong1993) [reply pkg/runtime/runtime.go]: fixed

**Review by berndverst — CHANGES_REQUESTED** [review]
Run `make modtidy-all` and commit the changes.

**Comment by zcong1993** [comment]
> Run `make modtidy-all` and commit the changes.

done

**Review by yaron2 — APPROVED** [review]
LGTM

**Pull request merged**